src/bp/core/config/config-loader.ts

```diff
@@ -61,6 +64,22 @@ export class ConfigProvider {
       config.pro.licenseKey = process.env.BP_LICENSE_KEY || config.pro.licenseKey
     }
 
+     [
+      ['BP_PORT', 'HTTTPSERVER.PORT'],
+      ['PORT', 'HTTTPSERVER.PORT'],
+      ['BP_HOST', 'HTTTPSERVER.PORT'],
+      ['BP_PROXY', 'HTTTPSERVER.PROXY'],
+      ['BP_LICENSE_KEY', 'PRO.LICENSEKEY']
```

Comment:
I think the checklist on the admin panel also needs to be updated to include those changes